CHANGELOG

```diff
@@ -1,5 +1,18 @@
 CHANGELOG
 
+UNRELEASED
+  * FEATURE : add AlgoliaSearchHelper.getState(filters) to get a filtered state
+  * FEATURE : add AlgoliaSearchHelper.getStateAsQueryString to get a part of
+  the state as a queryString
+  * FEATURE : add AlgoliaSearchHelper.setStateFromQueryString to set the state
+  of the helper with a query string
+  * FEATURE : add AlgoliaSearchHelper.getConfigurationFromQueryString. Reads
+  a query string and returns the associated partial helper configuration
+  * FEATURE : add AlgoliaSearchHelper.getForeignConfigurationInQueryString to
+  get the configuration contained in a qs that is not from the helper
+  * FIX : all the properties that are supposed to be numbers are now parsed in
```

Comment:
Consistency. We were able to have them in both format (string and numbers) but when deserializing would always use strings. Therefore we had inconsistencies when testing with numbers as input and we wanted to compare the results of the values.

This way we are guaranteed that the numeric will be numeric.